openlibrary/plugins/openlibrary/js/cover_add_manage.js

```diff
@@ -0,0 +1,58 @@
+export default function(){
+    // page may not be loaded via iframe
+    if (parent && parent.closeThrobber) {
+        parent.closeThrobber();
+    }
+    $('#form.addcover-form').submit(function(event) {
+        var file;
+        var url;
+        var coverid;
+        function val(selector) {
```

Comment:
perhaps this function could be moved out the closure to the top of the file as it doesn't require anything inside it.